core/src/com/unciv/ui/cityscreen/SpecialistAllocationTable.kt

```diff
@@ -14,7 +15,18 @@ class SpecialistAllocationTable(val cityScreen: CityScreen): Table(BaseScreen.sk
 
     fun update() {
         clear()
-
+        // Auto/Manual Specialists Toggle
+        // Color of "color" coming from Skin.json that's loaded into BaseScreen
+        // 5 columns: unassignButton, AllocationTable, assignButton, SeparatorVertical, SpecialistsStatsTabe
+        if(cityInfo.manualSpecialists) {
+            val manualSpecialists = "Manual Specialists".tr().toLabel().addBorder(5f, BaseScreen.skin.get("color", Color::class.java))
```

Comment:
`.toLabel()` already has an internal `.tr()` call, so no need to do it twice

Reply:
Similar to the little hint [in my comment 17 days back](https://github.com/yairm210/Unciv/pull/6650#issuecomment-1113937648)

Reply:
Yup